content/quickstart_ocaml_project_dune.md

```diff
@@ -4,7 +4,9 @@ tags: [learning, quickstart]
 
 # Quickstart an OCaml app project using Dune
 
-(adapted from @bobbypriambodo's post [here](https://medium.com/@bobbypriambodo/starting-an-ocaml-app-project-using-dune-d4f74e291de8))
+(adapted from @bobbypriambodo's post [here](https://medium.com/@bobbypriambodo/starting-an-ocaml-app-project-using-dune-d4f74e291de8) )
```

Comment:
Why do we want an extra space here? What good does it do?

Reply:
You can delete it. I first put the date or the article there then moved it, letting this useless space.

Reply:
I can't delete it, it's in your pull request. Update your branch and force push.

Reply:
Done